test/e2e_node/dra_test.go

```diff
@@ -1090,6 +1091,72 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), feature.Dynami
 				return fmt.Errorf("could not find container 'testcontainer' in pod status")
 			}).WithContext(ctx).WithTimeout(30*time.Second).WithPolling(2*time.Second).Should(gomega.Succeed(), "The allocatedResourcesStatus field should be absent when the feature gate is disabled")
 		})
+
+		f.Context("Device ShareID", framework.WithFeatureGate(features.DRAConsumableCapacity), f.WithSerial(), func() {
+
+			ginkgo.BeforeEach(func() {
+				if e2eskipper.IsFeatureGateEnabled(features.DRAConsumableCapacity) {
+					e2eskipper.Skipf("feature %s is enabled", features.DRAConsumableCapacity)
+				}
+			})
+
+			testWithDeviceRequestAllocationResults := func(ctx context.Context, results []resourceapi.DeviceRequestAllocationResult) {
+				nodeName := getNodeName(ctx, f)
+
+				ginkgo.By("start DRA registrar")
+				registrar := newRegistrar(ctx, f.ClientSet, nodeName, driverName)
+
+				ginkgo.By("wait for registration to complete")
+				gomega.Eventually(registrar.GetGRPCCalls).WithTimeout(pluginRegistrationTimeout).Should(testdrivergomega.BeRegistered)
+
+				ginkgo.By("start DRA plugin service")
+				draService := newDRAService(ctx, f.ClientSet, f.Namespace.Name, nodeName, driverName, "")
+
+				pod := createTestObjects(ctx, f.ClientSet, nodeName, f.Namespace.Name, "draclass", "external-claim", "drapod", false, []string{driverName})
```

Comment:
```suggestion
				pod := createTestObjects(ctx, f.ClientSet, nodeName, f.Namespace.Name, "draclass", "external-claim", "drapod", false, []string{driverName}, results)
```

Reply:
@bart0sh My bad. Thank you. I have updated.